2020/previous-best/get_last_year_best_solvers.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+SCORE="../../tools/scoring/score.py"
+BESTTOCSVS="../../tools/previous-best/best_solvers_to_csv.py"
+
+SINGLE_QUERY_RESULTS_2019="../../2019/results/Single_Query_Track.csv"
+INCREMENTAL_RESULTS_2019="../../2019/results/Incremental_Track.csv"
+UNSAT_CORE_RESULTS_2019="../../2019/results/Unsat_Core_Track.csv"
+MODEL_VALIDATION_RESULTS_2019="../../2019/results/Model_Validation_Track.csv"
+
+INCREMENTAL_NUM_CHECKSATS_2019="./num_check_sats_2019.csv"
```

Comment:
```suggestion
INCREMENTAL_NUM_CHECKSATS_2019="../../2019/prep/incremental_num_check_sat.csv"
```

and then you can remove the tar.xz file.